src/routes/users.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+
+let users = [];
+
+router.get('/', (req, res) => {
+  res.json(users);
+});
+
+router.post('/', (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.status(400);
+    res.send('User is not added');
+
+    return;
+  }
+
+  const newUser = {
+    id: Math.floor(Math.random() * 100),
+    name,
+  };
```

Comment:
Check @naziamoff comment about the layouts.

Create 3 layouts. Routes, Controllers, Services.

For example:

**routes/users.js**

`router.post('/', userController.createUser);`

**controller/users.js**

```
const createUser = (req, res) => {
  const { name } = req.body;

  if (!name) {
    res.status(400);
    res.send('User is not added');

    return;
  }
  const newUser = userService.createUser(name);

  res.statusCode = 201;
  res.json(newUser);
}
```

And in service, you need to add the logic of interaction with the database.
Same in all other cases